core/templates/core/base.html

```diff
@@ -95,12 +95,12 @@
 								Account
 							</a>
 							<div class="dropdown-menu dropdown-menu-right greenify-dropdown" aria-labelledby="navbarDropdownMenuLink">
```

Comment:
We could possibly populate this dropdown using the config as well in a future PR.